src/components/Book.js

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable react/button-has-type */
+/* eslint-disable jsx-a11y/anchor-is-valid */
+/* eslint-disable react/destructuring-assignment */
+import React, { Component } from 'react';
+
+class Book extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      // eslint-disable-next-line react/prop-types
+      genre: props.genre,
+      // eslint-disable-next-line react/prop-types
+      title: props.title,
+      // eslint-disable-next-line react/prop-types
+      author: props.author,
+      // eslint-disable-next-line react/no-unused-state
+      comments: [],
+      progress: 0,
+      currentChapter: 1,
+    };
+  }
+
+  handleComment = () => {
+    // code to handle adding comments
+  }
+
+  handleRemove = () => {
+    // code to remove the book
+  }
+
+  handleEdit = () => {
+    // code to edit the book
+  }
+
+  handleLoad = () => {
+    // code to load the progress of the book
+  }
+
+  render() {
+    return (
+      <div className="book-container">
+        <div>
+          <div>
+            <p>
+              Genre:
+              {this.state.genre}
+            </p>
+            <p>
+              Title:
+              {this.state.title}
+            </p>
+            <p>
+              Author:
+              {this.state.author}
```

Comment:
- [ ] Kindly note that the title and the author of the book should come from the props and not in the state of the actual component.